examples/sagemaker-tensorflow/README.md

```diff
@@ -0,0 +1,44 @@
+# Training and Serving Merlin on AWS SageMaker
+
```

Comment:
All these are nits and up to you:

// Removing conjugations of the verb "to be" often are more straightforward and clear.
...package your own models that you can train and deploy in the SageMaker environment.

// Using present tense instead of future tense makes your voice more authoritative to the reader.
The following notebook shows how to use...